site.deploy.yml

```diff
@@ -65,24 +86,30 @@
         - lamp_installed.changed
         - ansible_deployed.changed
 
+# If we haven't failed by this point, we're ready to prep for
+# service and WordPress deployment.
 - name: Deploy WordPress to defined hosts in the 'wordpress' group
   any_errors_fatal: true
   gather_facts: false
   hosts: wordpress
 
+  # Create '/etc/ansible'.
```

Comment:
This comment is in the wrong spot and should be above the task itself, though as it says the same thing as the `name:` of the task I think it would be best to remove it.